### ISSUE TITLE
Move cursor with arrows (keyboard)

### ISSUE THREAD

**Issue by zenturacp** [flameshot-org/flameshot#3141]
### Feature Description

In greenshot you can move the cursor with the keyboard to adjust the cross - that's very nice when doing region snaps

**Comment by gstarch** [comment]
@zenturacp Is what you are asking for different than the current Resize/Move Selection by 1px, which has keyboard arrow (or SHIFT + arrow) shortcuts by default?

**Comment by zenturacp** [comment]
Yes exactly @gstarch - on lowres no problem but on highres i need to crop the picture afterwards if there is 1px line too much etc.. I simply cant see it on the monitor if im 1px off.. 

In greenshot the magnification also shows the selection and then the cursor moves the magnify glass when im exactly where i want to be i can just hit the mouse and move to other point and then arrange with keyboard and release ..

Indeed very very acurate

**Comment by mmahmoudian** [comment]
I don't know how to combine the magnification with the keyboard movement. At the moment we have also this magnifier which let you have pixel-by-pixel movement. I don't have high-res monitor to see the experience, hence I'm asking if this helps. You can activate it via:

![image](https://user-images.githubusercontent.com/390889/234281044-88051184-28a9-4732-84e4-167bc28c4866.png)

**Comment by zenturacp** [comment]
Exaclty i use it - but it's quite time consuming to move it and the mouse it not like really good at precission.. 

In greenshot its so much easier because 1 tap at the arrow up moves up 1 pixel and holding just moves the cursor continuos, then if i need to reference some point up i just hold the mouse and just press they keyboard..

Please try and install greenshot and test it - if possible, its very easy to go to the exact point / corner of a button etc. also the magnifyer is bigger so its easier to see a bigger picture of the area.

**Comment by zenturacp** [comment]
> Bump. I would also like to see this feature, it would make it much easier to take an accurate screenshot as described by zenturacp above. When Windows was my main OS, I used ShareX to take screenshots and it had this feature. And I'd like to reiterate that just having the magnifier isn't enough, but _combining_ it with the arrow key functionality to move the region pixel by pixel makes the process a breeze.

Are you still using Windows? I did the change in a fork and tested it - and it works, but have only build for Windows to test, it was a mess so need to redo it again, and make new binaries

**Comment by plpagadorfdez** [comment]
I like @Igetin approach, maybe because I have also used Greenshot in Windows and that's similar to how it works. I was planning to give it a try like that, but I think there's one scenario that it's not covered in those three bulletpoints:

**When a screenshot area has already been selected but user wants to start a new selection**, in this case Flameshot wouldn't allow the user to move the mouse cursor by 1px since pressing arrow keys would move the existing selection instead.

I saw that in the shortcuts' configuration panel there's still room for `Ctrl+[arrow key]`, so another solution could be using that combination to move mouse cursor by 1px instead of the selected area. It would go like this:

**When dragging the corner of an existing selection** (by holding the left mouse button on it), the shortcut `Ctrl+[arrow key]` should move by 1px in the corresponding direction:
* The mouse cursor.
* The magnifier view when it's enabled.
* That vertex of the rectangle, effectively updating the size of the selected area.

**In any other scenario**, the shortcut `Ctrl+[arrow key]` should move by 1px in the corresponding direction:
* The mouse cursor.
* The magnifier view when it's enabled.

With this implementation, Flameshot would work identically for users that don't need or are not used to this new feature, while allowing to have it for those (like me) who really miss it.
